### DISCUSSION TITLE
Join us and offer help us offer Official Docker Support

### DISCUSSION THREAD

**Discussion by PeerRich** [calcom/docker#100] [Announcements]
Hey Community. I know the docker file has had it's fair amount of problems and since no one in our team has any Docker experience, we have a hard time fixing those pesky bugs.

However, with our recent round of funding (https://cal.com/blog/cal-v-1-5) we're looking to bring the Docker repository in-house and officiall support it. 

If you have a lot of Docker experience, we'd love to hire you either as a consultant, part-time or even full-time if you have other engineering skills (frontend/backend)

**Comment by PeerRich** [comment]
Once we start offering official support, we should also move calcom/docker into calcom/cal.com into /apps/docker and close this repository

**Reply by AmoghSaxena** [reply]
Awesome!

**Comment by lavakush07** [comment]
Hello PeerRich , I would love to work as part-time for this . I am good with docker,kubernetes and few devops tools. I am also a MERN stack web developer.

**Comment by carsonbird** [comment]
@PeerRich Well, I'm tired of troubleshooting a new issue during build time every time a version is released. Can I help? Calcom is the best offering out there, from what I've seen, and I'd love to reliably use it containerized in a production environment.

**Comment by TauqeerAhmad5201** [comment]
Hey @PeerRich  
Is there any workaround of composing or building the Docker releases? I've gotten some experience as well. Would love to contribute.
